src/test/java/uk/gov/pay/directdebit/payers/services/PayerServiceTest.java

```diff
@@ -22,6 +23,8 @@
 
 import static org.hamcrest.core.Is.is;
 import static org.junit.Assert.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
```

Comment:
Wild thing, you make my heart sing
You make everything groovy, wild thing
Wild thing, I think I love you
But I wanna know for sure
Because I think the GDS Way Java styleguide says we should avoid wildcard imports

Reply:
<3